communications/meetings.md

```diff
@@ -0,0 +1,67 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+
+- Determine the scope and minimum length of the meeting.
+- Give all participants as much context as possible to make an informed decision on RSVP.
+- Gather topics and issues for agenda items that are relevant to the scope of the meeting,
+done either individually or through crowdsourcing to increase accuracy and minimize bias
+- Asynchronously resolve as many agenda items as possible before the meeting.
+
+Guide:
+
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+- Create the meeting agenda on Hugo for the meeting, provide sufficient time ahead of the meeting to accommodate all participants.
+    - For a net-new meeting, the organizer(s) should define the initial scope of agenda items.
+    - For a recurring meeting with past sessions, use the agenda/notes items from the previous meeting session as a starting point. Remove ones no longer needed or already resolved.
+- Initiate one or more meeting note(s) on Hugo for meeting agenda discussions.
```

Comment:
Would recommend saying 'initiating one note for meeting agenda' to avoid confusion of meeting note.

Reply:
but that's the Note feature regardless, as Agenda on Hugo cannot be sync'ed to Slack as thread there.

the one or more is intended to accommodate for meetings with complex scopes, and breaking down notes per-item or big topic would be benefitial.

this confusion may need a screenshot/gif/video to iron out, as one sentence really doesn't serve the purpose

Reply:
ahhh then I think the title of each note will be important. If you set an agenda like:
Agenda
- Facebook Connector
- LocusML

Then initiate a note for each agenda item titled: 
Agenda Item Discussion: Facebook Connector
- Updates:
- Issues:

Something like that to be very clear

Reply:
absolutely. the title portion can be very useful.